lua/tests/current_search_paths_spec.lua

```diff
@@ -0,0 +1,24 @@
+local lean = require('lean')
+local helpers = require('tests.helpers')
+
+require('tests.helpers').setup {
+  lsp = { enable = true },
+  lsp3 = { enable = true },
+}
+describe('lean.current_search_paths', function()
+  describe('lean3', function()
+    it('returns the current search paths', function()
+      vim.api.nvim_command("edit lua/tests/fixtures/example-lean3-project/test.lean")
+      helpers.wait_for_ready_lsp()
+      assert.is.truthy(lean.current_search_paths())
```

Comment:
Could we have this check that the particular path returned is correct? And can we also have a test for nested files?

Reply:
Yes! Good points, this was waiting for neovim/neovim#15058 to be fixed but will likely get back to it today or tomorrow.